scripts/hooks/nucypher_dkg.py

```diff
@@ -224,7 +224,11 @@ def get_publication_endpoint(self) -> str:
     domain=network,
     registry=registry,
     known_nodes=[
-        Ursula.from_teacher_uri(f"https://{network}.nucypher.network:9151", min_stake=0)
+        Ursula.from_teacher_uri(
```

Comment:
I think we may have to adjust the parameters for this hook. It may need an additional polygon endpoint uri for the Coordinator agent - see https://github.com/nucypher/nucypher/pull/3137/files#diff-f7222caf9922de9891fb48e5d4bb493e3e2f960bd4438409747796dec087add8R105.

Of course we can do this in a separate/follow-up PR.